Library/Formula/tomcat-runner.rb

```diff
@@ -0,0 +1,14 @@
+class TomcatRunner < Formula
+  desc "Run Tomcat without an installed distribution"
+  homepage "https://github.com/jsimone/webapp-runner"
+  url "http://repo1.maven.org/maven2/com/github/jsimone/webapp-runner/8.0.24.0/webapp-runner-8.0.24.0.jar"
+  version "8.0.24.0"
```

Comment:
yes, `brew create <jar_url>` correctly identified the version number, I guess the `url` string could be interpolated with `#{version}` to avoid duplication.